locust_tasks/locustfile.py

```diff
@@ -166,49 +189,163 @@ def correct_address(self):
 The simulated user steps through the pages one by one. They don't go down any of the 
 correction/error paths as this doesn't trigger any significant server side work.
 """
-class request_new_code(SequentialTaskSet):
+class RequestNewCodeSMS(SequentialTaskSet):
+    """
+    Class to represent a user requesting a new UAC, which is to be sent by SMS.
+    """
 
-    # All users arrive at the start page
     @task(1)
     def start_page(self):
-        self.client.get("/en/start/")
+        """
+        GET Start page
+        """
+        self.case = get_next_case()
+        with self.client.get('/en/start/', catch_response=True) as response:
+            verify_response('RequestUacSms-Start', self, response, 200, Page.START)
         
     @task(2)
-    def request_new_access_code(self):
-        self.client.get("/request-access-code")
+    def enter_postcode(self):
+        """
+        POST postcode
+        """
+        with self.client.post("/en/requests/access-code/enter-address/", {
+            'form-enter-address-postcode': self.case['postcode']
+        }, catch_response=True) as response:
+            self.address_to_select = extractAddress(response, self.case["uprn"])
```

Comment:
'extractAddress' function name is not accurate. It's not extracting an address. It's getting the id of the radio button for specified UPRN

Reply:
I'll look at that

Reply:
Basically the function and any comments need the description to be more accurate.

Reply:
ok

Reply:
I've had a look at this. The thing that it returns is the JSON containing the UPRN and address e.g. {"uprn": "200002986059", "address": "6 Moretons Close, Whittlesey, Peterborough, PE7 1XP"}
So I'm happy to rename it extractJSON or something like that?

Reply:
How about something like extractAddressRadioButtonValue().
If that is too long then perhaps extractAddressValue(), with a function level comment to explain it's purpose in more detail (as the name isn't quite enough to summarise what it is doing).

Reply:
I'm happy to name it extractAddressRadioButtonValue() as it only gets called once in each task sequence.

Reply:
done and pushed